.travis.yml

```diff
@@ -7,9 +7,16 @@ install: true
 script:
   - |
     if [[ $TRAVIS_REPO_SLUG == "pinterest/ktlint" && $TRAVIS_BRANCH == "master" && $TRAVIS_PULL_REQUEST == "false" ]]; then
-      ./gradlew build && ./mvnw -Ddeploy=maven-central -DskipStaging=true -Dgpg.skip=true && ./mvnw -Ddeploy=github -Dgpg.skip=true -Dgithub.draft=true -Dgithub.description="https://github.com/pinterest/ktlint#access-to-the-latest-master-snapshot"
+      # Gradle
+      ./gradlew build
+      # Maven
+      ./mvnw -Ddeploy=maven-central -DskipStaging=true -Dgpg.skip=true && ./mvnw -Ddeploy=github -Dgpg.skip=true -Dgithub.draft=true -Dgithub.description="https://github.com/pinterest/ktlint#access-to-the-latest-master-snapshot"
     else
-      ./mvnw clean verify
+      # Gradle
+      ./gradlew build # Make sure it builds on each PR
+      ./gradlew ktlint || true # Return true for now until we have fixed all of the ktlint errors
```

Comment:
Actually maven already runs ktlint on this project (sorry, missed that and opened #463), so `|| true # Return true for now until we have fixed all of the ktlint errors` should be not needed here.

Reply:
Check this line: https://github.com/pinterest/ktlint/blob/b09945530513a6bf4edf7c5f45524609c74b26ce/ktlint/pom.xml#L199

Reply:
If it does run, why are there no errors? Look at the TravisCI output of this build, there are many formatting errors.

Reply:
It doesn't use `--android` flag.

Reply:
Removed.